tests/test_utils.py

```diff
@@ -40,6 +81,28 @@ def test_makes_request(self, mock_manager):
         mock_manager.assert_called_once_with(num_pools=1)
         mock_connection.request.assert_called_once()
 
+        with patch.dict(os.environ, {'http_proxy': ''}):
+            utils.make_request(
+                'GET',
+                host,
+                url,
+                'a_user',
+                'a_pass'
+            )
+            mock_manager.assert_called_with(num_pools=1)
+            self.assertEqual(mock_connection.request.call_count, 2)
```

Comment:
I suggest a comment that explains why this test exists, e.g. `# In case the proxy variable exists but is empty, it shouldn't break things`.